eng/pipelines/helix.yml

```diff
@@ -29,6 +30,7 @@ steps:
             /p:IsExternal=${{ parameters.isExternal }}
             /p:Creator=${{ parameters.creator }}
             /p:OfficialBuildId=${{ parameters.officialBuildId }}
+            /p:ContinuousIntegrationBuild=true
```

Comment:
Why is this needed at all?

Reply:
This is needed so Arcade correctly generates the version build variables that I use in the blob storage name: $(PackageVersion) and $(VersionSuffix)

Reply:
Really, when sending jobs to helix? I didn't see any change in sendtohelix.proj that uses those variables. I did actually see it for publish.proj, which makes sense.

Reply:
Sorry, you are right. It isn't needed here anymore.

Reply:
Btw, I opened: https://github.com/dotnet/arcade/issues/2073 since -ci flag should set that msbuild property.